seb_openedx/admin.py

```diff
@@ -1,7 +1,27 @@
 # -*- coding: utf-8 -*-
 """ Admin.py """
 from django.contrib import admin
+from django.db.models import TextField
+from seb_openedx.forms import SebCourseConfigurationForm
+from seb_openedx.models import (
+    ForbiddenCourseAccess,
+    SebCourseConfiguration,
+)
+from seb_openedx.widgets import ListWidget
+
+
+class SebCourseConfigurationAdmin(admin.ModelAdmin):
+    """Admin for the SebCourseConfiguration model."""
+    list_display = [
+        'course_id',
+        'user_banning_enabled',
+    ]
+    search_fields = ('course_id', )
```

Comment:
we should also search by the keys

Reply:
Done in method get_search_results in https://github.com/eduNEXT/seb-openedx/pull/39/files#diff-d5de91ba936faae76d26f21daa0372c7R35